nightly.yml

```diff
@@ -16,10 +16,16 @@
       inputs:
         version: '2.1.x'
     # CodeSign
-    - task: EsrpCodeSigning@1
+    - task: EsrpCodeSigning@5
       displayName: 'ESRP CodeSign - OData'
       inputs:
         ConnectedServiceName: 'ESRP CodeSigning - OData'
+        AppRegistrationClientId: 'ce6e119c-a8c2-4744-9699-cd5389102d0d'
+        AppRegistrationTenantId: '33e01921-4d64-4f8c-a055-5bdaffd5e33d'
+        AuthAKVName: ODataESRPKeyVault
+        AuthCertName: ODataCodeSignAuth-AutoRotate
+        AuthSignCertName: ODataCodeSigningCert2024
+        ServiceEndpointUrl: 'https://api.esrp.microsoft.com/api/v2'

```

Comment:
Should this information be exposed like this?

Reply:
@habbes Use variables

Reply:
@gathogojr @ElizabethOkerio I've updated the code to use a variable group.